azure_functions_worker/mmap_handler/file_accessor_factory.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+from .file_accessor_linux import FileAccessorLinux
+from .file_accessor_windows import FileAccessorWindows
+
+
+class FileAccessorFactory:
+    @staticmethod
+    def create_file_accessor():
+        if os.name == 'posix':
+            return FileAccessorLinux()
+        else:
```

Comment:
Should we check for the windows os.name and have an else for everything unknown?

Reply:
Yes, following the same convention as in the host (check for Windows, all others go to Unix.